src/js/components/Notification/index.d.ts

```diff
@@ -8,6 +9,7 @@ export interface NotificationProps {
   status?: StatusType;
   toast?: boolean;
   onClose?: (...args: any[]) => any;
+  icon?: Icon;
```

Comment:
can we actually change this to then take out the import above we usually use this pattern. 
`icon?: JSX.Element;`

Reply:
Yes, followed the pattern and refactored.